substrate/tendermint/primitives/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "sp-tendermint"
+version = "0.1.0"
+description = "Tendermint primitives for Substrate"
+license = "AGPL-3.0-only"
+repository = "https://github.com/serai-dex/serai/tree/develop/substrate/tendermint/primitives"
+authors = ["Luke Parker <lukeparker5132@gmail.com>"]
+edition = "2021"
```

Comment:
2022

Reply:
There is no Rust 2022 edition. https://doc.rust-lang.org/edition-guide/editions/index.html